buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -756,7 +756,11 @@ class BeamModulePlugin implements Plugin<Project> {
       project.checkstyle { toolVersion = "8.7" }
 
       // Configures javadoc plugin and ensure check runs javadoc.
-      project.tasks.withType(Javadoc) { options.encoding = 'UTF-8' }
+      project.tasks.withType(Javadoc) {
+        options.encoding = 'UTF-8'
+        options.addStringOption('Xdoclint:all', '-quiet')
```

Comment:
Does replacing `'-quiet'` with `true` work?

ref: https://discuss.gradle.org/t/how-to-send-x-option-to-javadoc/23384/5

Reply:
Yes, an `options.addBooleanOption('Xdoclint:all', true)` will have the same effect on the linter.